backends/source/dppl_sycl_platform_interface.cpp

```diff
@@ -148,7 +148,12 @@ size_t DPPLPlatform_GetNumNonHostPlatforms ()
 
 size_t DPPLPlatform_GetNumNonHostBackends ()
 {
-    return get_set_of_non_hostbackends().size();
+    auto be_set = get_set_of_non_hostbackends();
```

Comment:
What was this problem?

Reply:
Non-void function does not return value

Reply:
`return get_set_of_non_hostbackends().size();` does not return value?

Reply:
The value should be returned, but at this point klocwork shows a warning

Reply:
Weird that klockwork requires this change, but if it fixes the warning then I am OK with this change.